ui/revenuecatui/src/test/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableProcessorTest.kt

```diff
@@ -81,7 +81,12 @@ class VariableProcessorTest {
 
     @Test
     fun `process variables processes price_per_period`() {
-        expectVariablesResult("{{ price_per_period }}", "PRICE_PER_PERIOD")
+        expectVariablesResult("{{ price_per_period }}", "$67.99/yr")
+    }
+
+    @Test
+    fun `process variables processes price_per_period localized in spanish`() {
+        expectVariablesResult("{{ price_per_period }}", "$67.99/a", esLocale)
```

Comment:
Honestly, I don't like this abbreviation that much 🤔. IMO it may be even more confusing. I'm wondering whether we should use the full word, or just have our own localization for this... On the other hand, this is what the android formatter does, so might be ok...

Reply:
Have you checked the other `FormatWidth`s? If you look at iOS `Localization.abbreviatedUnitLocalizedString` I have some heuristics to get an abbreviation that matches an expected length based on `unitAbbreviationLengthPriorities`.

You can iterate on this in a follow-up PR, but that's how I managed to get desirable abbreviations in different languages.

Reply:
I did check and in spanish at least it went from `1 a.` to `1 año`. There was nothing in between... But yeah, we can improve these as you did in iOS later.